src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -256,9 +255,7 @@ private IOperation CreateInternal(BoundNode boundNode)
                     return CreateBoundIsPatternExpressionOperation((BoundIsPatternExpression)boundNode);
```

Comment:
Please add tests that expose problems with the current code.

---
In reply to: [135119021](https://github.com/dotnet/roslyn/pull/21407#discussion_r135119021) [](ancestors = 135119021)